apps/antalmanac/src/components/Calendar/CalendarToolbar.tsx

```diff
@@ -300,6 +301,7 @@ function CalendarPaneToolbar(props: CalendarPaneToolbarProps) {
                 alignItems: 'center',
                 padding: 1,
                 borderRadius: '4px 4px 0 0',
+                backgroundColor: isDark ? 'rgb(24, 26, 27)' : '',
```

Comment:
Could you store this as a variable somewhere? I think it would be helpful to keep theme colors centralized somewhere visible. The best place would be on the theme object where, and the next best class is just a top-level constant. e.g.

```tsx
const DARK_CALENDAR_BACKGROUND = 'rgb(25, 26, 27)'
```

Also, maybe hex color is nicer? I think HSL > Hex > RGB is a good, arbitrary preference to have. OKLCH is interesting and apparently displays colors more accurately, but I don't have experience with it.